### PR TITLE
config circle ci for deployment

### PR DESCRIPTION
**Pull request by jollyjoker992**
- Config CI for build and deploy latest HTML code to repo [bitmark-inc.github.io](https://github.com/bitmark-inc/bitmark-inc.github.io)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,14 +3,13 @@
 # Check https://circleci.com/docs/2.0/language-go/ for more details
 version: 2
 jobs:
-  build:
+  test:
     docker:
-      # specify the version
```
Comment (araujobsd) [inline .circleci/config.yml:8]: Gratuitous change, here we specify the Golang version and I have add the comment for a reason.

File: .circleci/config.yml [hunk 2]
```diff
@@ -3,14 +3,13 @@
...
       - image: circleci/golang:1.12
         environment:
           - GO111MODULE: "on"
 
-    working_directory: ~/docs-testing/
```
Comment (araujobsd) [inline .circleci/config.yml:13]: Another gratuitous change that should not be part of this diff.

**Review by araujobsd — CHANGES_REQUESTED** [review]
This will never work!! igor will fail in master branch and the deployment will never run.
Or you add a logic to don't run igor on master branch and only run in branches, or all the issues must be fixed. 

A third option is to remove "igor" and let the "broken English" go wild.

File: .circleci/config.yml (hunk 1 above)
Comment (jollyjoker992) [reply .circleci/config.yml:8]: It's super common to get the meaning of that line. That's why i remove it. :)

File: tools/check-spelling.sh [hunk 3]
```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+# spelling check for markdown files
+dict='en_US,en_GB,en_BITMARK'
+printf -- 'file: %s\n' "${1}"
+awk '
+    BEGIN {
+        code=0
+        flag=1
+    }
+    /^~~~|^ *```/{
```
Comment (nodestory) [inline tools/check-spelling.sh]: @hxw I added another pattern here. Some code sections are wrapped by backticks.

Comment (hxw) [reply tools/check-spelling.sh]: [[:space:]]*

**Review by hxw — APPROVED** [review]

**Pull request merged**